### PR TITLE
Add ability to set seperate DB for test, development, and production

### PR DESCRIPTION
**Pull request by markmeyerphoto**
Adds the ability to set which database will be used for different environments. The default settings will look for a 'courtbotdb_test' database to use when running tests. This prevents tests from nuking the development DB when run.

It will require adding the the test db.

### CODE REVIEW

File: utils/db/db_connections [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+				connection: process.env.DATABASE_TEST_URL,
+				pool: {
+					afterCreate: function(connection, callback) {
+						connection.query("SET TIME ZONE 'UTC';", function(err) {
+							callback(err, connection);
+						});
+					}
+				}
+			}
+}
```
Comment (mfrederickson) [inline utils/db/db_connections]: Did you want to clean up the spacing before merging?

Comment (markmeyerphoto) [reply utils/db/db_connections]: Yeah, there's also some tabs in there and the file is named incorrectly. I'll fix it up.

Comment (markmeyerphoto) [reply utils/db/db_connections]: Should be better now. Thanks @mfrederickson.

Comment (mfrederickson) [reply utils/db/db_connections]: Thank you... I'm learning a lot by reading through the PRs and looking up stuff.

**Pull request merged**